tests/integration_tests/validation/conformance_suite_configurations/kvk_nt17.py

```diff
@@ -0,0 +1,66 @@
+import os
```

Comment:
Arbitrary line.

We've since implemented loading variation-level timing files (see `tests/resources/conformance_suite_timing`). Both local and GHA runs should be generating a `conf-kvk_nt17-timing.json` that you can rename and place in that folder to get shards optimized.